### PR TITLE
chore: auto hide card list when dragging a card

### PR DESCRIPTION
**Pull request by fundon**
If this behavior feels OK, add test later. 

https://user-images.githubusercontent.com/27926/225242810-a86a8713-0753-4a12-944b-0e76871ae2c6.mov

### CODE REVIEW

**Comment by doodlewind** [comment]
Looks good for mw, @joebeijing is this working as intended?

File: packages/blocks/src/components/block-hub.ts [hunk 1]
```diff
@@ -660,6 +664,7 @@ export class BlockHub extends NonShadowLitElement {
       data.type = affineType;
     }
     event.dataTransfer.setData('affine/block-hub', JSON.stringify(data));
+    // event.dataTransfer.setDragImage(blockHubElement, 0, 0);
```
Comment (doodlewind) [inline packages/blocks/src/components/block-hub.ts:667]: should be removed :)

**Comment by thorseraq** [comment]
The behavior LGTM! also confirmed with @joebeijing and @Svaney-ssman

> 

Confirmed with @joebeijing and @Svaney-ssman, they are open for this improment :)

**Review by doodlewind — APPROVED** [review]
Thanks for the improvement!

**Pull request merged**